build.sbt

```diff
@@ -49,7 +49,7 @@ lazy val all: Project = (project in file("."))
     name           := "mesmer-all",
     publish / skip := true
   )
-  .aggregate(extension, otelExtension, example, core, testkit)
+  .aggregate(extension, otelExtension, core, testkit, exampleAkka, exampleAkkaStream, exampleZio)
```

Comment:
Yes! Finally someone had a reason to split the examples to separate modules. Thanks! :)